spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/ops/LessSpec.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn.ops
+
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.T
+import org.scalatest.{FlatSpec, Matchers}
+
+class LessSpec extends FlatSpec with Matchers {
+  "Less Float operation" should "works correctly" in {
+    val input =
+      T(
+        Tensor[Float](T(1f, 2f, 2f)),
+        Tensor[Float](T(2f, 3f, 2f))
+      )
+
+    val expectOutput = Tensor[Boolean](T(true, true, false))
+
+    val output = Less[Boolean]().forward(input)
+    output should be(expectOutput)
+  }
+
+  "Less Double operation" should "works correctly" in {
+    val input =
+      T(
+        Tensor[Double](T(4.0, 3.0, 1.0)),
+        Tensor[Double](T(2.0, 3.0, 2.0))
+      )
+
+    val expectOutput = Tensor[Boolean](T(false, false, true))
+
+    val output = Less[Boolean]().forward(input)
```

Comment:
What do you mean of Less[Boolean]()? The T of Layer[T] is parameter type? So do you mean the layer has a boolean parameter?